doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -54,7 +120,7 @@ Figure 1-1 below shows this at a high level.
 <p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
 
-### Other thoughts
+### More thoughts
 - Kubernetes / CaaS should be recognised in ETSI NFV / MANO
 - It is feasible that Kubernetes, using operators and custom resources, could become a generic VNFM (i.e. the Kubernetes parts are the underlying engine, with the operators and custom resources being the equivalent of the specific VNFM).
 - Should each operator decide how Kubernetes is shared (if at all)?  Should we/someone suggest some best practice (such as, link Kubernetes cluster lifecycle to application lifecycle - so VNFv1.0 is in one cluster, VNFv2.0 is in another cluster. Are clusters shared between VNF vendors, or VNF types?).
```

Comment:
Having a K8s instalce per VNF is a bit of an overkill from overhead perspective. (Perhaps a necessary overkill for some, but still). I believe a reasonable recommendation could be a K8s cluster per vendor, or perhaps one per security domain. 

K8s domains could be shared between VNF vendors, but I feel that is out of the comfort zone of many.

Reply:
In general, there will be multiple k8s clusters for operational reasons, e.g. limiting the failure domain or providing (hard) multi-tenancy. Whether a SP runs different vendors' VNFs in the same or segregated clusters is a matter of policy. I don't think it's necessary for us to define that.

Reply:
> In general, there will be multiple k8s clusters for operational reasons, e.g. limiting the failure domain or providing (hard) multi-tenancy. Whether a SP runs different vendors' VNFs in the same or segregated clusters is a matter of policy. I don't think it's necessary for us to define that.

Totally agree.  We need to address the need for at least one Kubernetes clusters managed by a **CaaS Manager** (multi-cluster management plane). I reckon that has similar requirements on virtualised resource management as is already defined for VNFM in ETSI, and then similar requirements on Kubernetes cluster LCM as VNFM has for VNF LCM in ETSI.

Reply:
One of the benefits of K8S and Helm is increased granularity over upgrades along with canary testing, etc, so there would absolutely be a drive to have VNF 1.0 reside in one cluster and have it gradually transformed, via granular K8S operations, into VNF 2.0.

Reply:
> In general, there will be multiple k8s clusters for operational reasons, e.g. limiting the failure domain or providing (hard) multi-tenancy. Whether a SP runs different vendors' VNFs in the same or segregated clusters is a matter of policy. I don't think it's necessary for us to define that.

I think we should highlight some of these reasons for multiple K8S clusters, as the technical issues behind these reasons will be addressed over time. One example is the security partitioning: while we may choose to use NFVi VMs as a resource boundary, CRIs are moving to incorporate significantly stronger security partitioning that will remove a major reason for multiple K8S clusters.

Reply:
Sure - I will add this into the document.